app/models/event.rb

```diff
@@ -6,6 +6,13 @@ class Event < ActiveRecord::Base
   validates :start_on, presence: true
   validates :end_on, presence: true
   validates :name, presence: true
+  validate :validate_end_on_before_start_on
+
+  def validate_end_on_before_start_on
+    if end_on && start_on
```

Comment:
Reading up a bit on this, there's no guaranteed order that validations will be run. I'll check how we handle this case in Tinkerbox. 😀